### ISSUE TITLE
Cannot upgrade 2.19.0 by Homebrew

### ISSUE THREAD

**Issue by ykws** [yonaskolb/XcodeGen#1043]
Hi, I tried to upgrade to [2.19.0](https://github.com/yonaskolb/XcodeGen/releases/tag/2.19.0) ​by Homebrew.

However, I cannot upgrade.

2.19.0 not supported to install by Homebrew?
I checked [formulae](https://formulae.brew.sh/formula/xcodegen) that version is 2.18.0 yet.

```
$ brew install xcodegen
Warning: xcodegen 2.18.0 is already installed and up-to-date.
To reinstall 2.18.0, run:
 ​brew reinstall xcodegen
$ brew reinstall xcodegen
==> Downloading https://homebrew.bintray.com/bottles/xcodegen-2.18.0.catalina.bottle.tar.gz
Already downloaded: 
==> Reinstalling xcodegen 
==> Pouring xcodegen-2.18.0.catalina.bottle.tar.gz
🍺  /usr/local/Cellar/xcodegen/2.18.0: 29 files, 9.7MB
```

**Comment by ykws** [comment]
Resolution: Use [Mint](https://github.com/yonaskolb/Mint)

If already installed XcodeGen by Homebrew, uninstall it.

```
$ brew uninstall xcodegen
```

Then install XcodeGen by Mint and succeeded in upgrading 2.19.0 🎉 

```
$ mint install xcodegen
🌱 Cloning xcodegen 2.19.0
🌱 Resolving package
🌱 Building package
🌱 Copying resources for XcodeGen: SettingPresets ...
🌱 Installed xcodegen 2.19.0
🌱 Linked xcodegen 2.19.0 to /usr/local/bin
```

**Comment by freddi-kit** [comment]
Let me confirm it, it looks 2.19.0 is tried to update to brew but looks still in progress: https://github.com/Homebrew/homebrew-core/pull/71781

**Comment by ykws** [comment]
@freddi-kit Thank you for reference!

OK, this issue blocked by #1033

**Comment by yonaskolb** [comment]
Have opened a new PR to Homebrew for 2.20.0 which simply requires Catalina https://github.com/Homebrew/homebrew-core/pull/74854

**Comment by freddi-kit** [comment]
Thank you!
